[winging-it-3] Winging it Episode 3 Show Notes

stacyk opened the pull request:
![](https://source.unsplash.com/featured/?cute,animal&asdfiiighjk)

## Description
Winging it Episode 3

## Steps to test/reproduce
Look at the posts page https://deploy-preview-436--oddleventy.netlify.app/blog/
Look at the Episode 3 page https://deploy-preview-436--oddleventy.netlify.app/2023/10/24/winging-it-03/


## Show me
![Screenshot 2023-10-24 at 10 31 37 AM](https://github.com/oddbird/oddleventy/assets/1581694/5ef4dcc5-5ae3-48dd-82d0-cc8249f3b022)
![Screenshot 2023-10-24 at 10 31 14 AM](https://github.com/oddbird/oddleventy/assets/1581694/5794e388-52ff-4b73-a97e-6af99de10046)

stacyk commented on content/blog/wingingit/winging-it-03.md:
```diff
@@ -0,0 +1,44 @@
+---
+title: Intuitive Web Design with OOUX
+episode: 3
+author:
```
@mirisuzanne @jamesnw @jerivas tagging all of you since this is a bit time sensitive, but  when I added a tag I get an error in the terminal (see screenshot). 

```
tags:
  - ooux 
  ```

  also tried:

```  
  tags:
  - design
 ```

![Screenshot 2023-10-24 at 9 55 19 AM](https://github.com/oddbird/oddleventy/assets/1581694/8e2bcb8b-c5fc-4394-8837-91a6fbf857c6)

stacyk commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Episode 03 - OOUX Unintuitive Objects](https://trello.com/c/EDlhk5pH/29-episode-03-ooux-unintuitive-objects)

jamesnw commented on content/blog/wingingit/winging-it-03.md:
```diff
@@ -0,0 +1,47 @@
+
+- [Denver Zoo](https://denverzoo.org/)
+- [OOUX Certification](http://partners.ooux.com/348125/17189)
+- [Slides](https://xd.adobe.com/view/59a38984-ff)
```
```suggestion
- [Slides](https://xd.adobe.com/view/59a38984-ff8c-4ae6-b09f-2cbf31f235fc-0cf0/)
```
The link is cut off, here's the one from Youtube

stacyk replied:
@jamesnw fixed in https://github.com/oddbird/oddleventy/pull/436/commits/169a5d6c1de86d698c2bfa7e00354c40863fb205. Thanks for catching that. I thought it was weird that it was so short. I should have known to test it.

jamesnw reviewed (approved):
🚀

stacyk commented:
> * In the byline, the link for David goes to David Glick, but should go to David Herron.

@SondraE good catch. updated in https://github.com/oddbird/oddleventy/pull/436/commits/169a5d6c1de86d698c2bfa7e00354c40863fb205

> * I briefly show and then talk about OddBird's Mastodon account, so we could add that link to the site and the youtube notes. https://front-end.social/@OddBird

added in same commit

> * Add a note to youtube and oddsite beside the OOUX Certification: "Affiliate link included"

I shortened it to just (affiliate link) in both places.

SondraE reviewed (approved):
ship it!
